src/js/player.js

```diff
@@ -83,6 +83,24 @@ const TECH_EVENTS_RETRIGGER = [
    */
   'abort',
 
+  /**
+   * Fired when the source is set on the tech causing the media element
+   * to reload.
+   *
+   * @private
+   * @method Player#handleTechSourceset_
```

Comment:
is this a holdover from before extracting out https://github.com/videojs/video.js/pull/4717?

Reply:
yeah

Reply:
and I just realized that this was for exposing the doc for it publicly. So, we just need to reword this one and remove the `@private` from another.